ui/desktop/app/components/settings-card/logs/index.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+
+export default class SettingsCardLogsComponent extends Component {
+  // =services
+  @service ipc;
+
+  // =attributes
+
+  get logLevels() {
+    return [
+      {
+        label: 'error',
+        value: 'error',
+      },
+      {
+        label: 'warn',
+        value: 'warn',
+      },
+      {
+        label: 'info',
+        value: 'info',
+      },
+      {
+        label: 'debug',
+        value: 'debug',
+      },
```

Comment:
since both key/value are the same here, should we just use an array?

Reply:
Yup good call, updated!